.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -88,13 +77,9 @@ body:
       multiple: true
       options:
         - "Yes I am willing to submit a PR!"
-  - type: dropdown
+  - type: markdown
```

Comment:
Is there a reason you are using two markdown items where the first only include <br>? can't you use a multiline markdown?

Reply:
Its a none mandatory selection - instead of a check box that translates to task
![image](https://user-images.githubusercontent.com/91742238/151699737-00b501ad-03b0-4ac1-ad73-7ad3468c8890.png)

Reply:
@yuval-hazaz